templates/components/common/currency-selector.html

```diff
@@ -5,16 +5,23 @@
         <ul class="dropdown-menu" id="currencySelection" data-dropdown-content aria-hidden="true" tabindex="-1">
             {{#each currency_selector.currencies}}
             <li class="dropdown-menu-item">
-                <a href="{{{switch_url}}}">
-                    {{#if is_active}}
-                    <strong>{{name}}</strong>
-                    {{else}}
-                    {{name}}
+                <a href="{{{switch_url}}}"
+                   {{#if ../currency_selector.has_discount}}
```

Comment:
what if they have discounts + GC?

Reply:
I was going to ask @bc-charlesho this tomorrow. The current implementation just shows discount warning

Reply:
Hi Matt, Cart Level Discounts going forwards will be marketed as Promotions. So we can say this:

"Promotions and gift certificates that don't apply to the new currency will be removed from your cart. Are you sure you want to continue?"